boards.txt

```diff
@@ -398,6 +398,94 @@ metro52840.menu.debug.l3.build.sysview_flags=-DCFG_SYSVIEW=1
 #
 # -------------------------------------------------------
 
+# ----------------------------------
+# Opossum Pouch
+# ----------------------------------
+op1.name=Opossum Pouch nRF52833 (op1)
+op1.bootloader.tool=bootburn
+
+# Upload
+op1.upload.tool=nrfutil
+op1.upload.protocol=nrfutil
+op1.upload.use_1200bps_touch=true
+op1.upload.wait_for_upload_port=true
+op1.upload.maximum_size=290816
+op1.upload.maximum_data_size=52224
+
+# Build
+op1.build.mcu=cortex-m4
+op1.build.f_cpu=64000000
+op1.build.board=NRF52833_op1
+op1.build.core=nRF5
+op1.build.variant=opossum_pouch_nrf52833
+op1.build.usb_manufacturer="Nordic"
+op1.build.usb_product="nRF52833"
+op1.build.extra_flags=-DNRF52833_XXAA {build.flags.usb}
+op1.build.ldscript=nrf52833_s140_v6.ld
+op1.build.vid=0x239A
+op1.build.pid=0x8029
```

Comment:
can you tell us more about this board. Also each board must has its own dedicated VID/PID and cannot re-use PID from another especially one is selling as commercial product.

Reply:
That's a mistake, Opossum Pouch is the board that I'm developing. I've got it defined on a private fork.